index.d.ts

```diff
@@ -1,8 +1,27 @@
 import { Middleware, MiddlewareAPI } from 'redux';
-import { Observable } from 'rxjs/Observable';
+import { Observable, ObservableInput } from 'rxjs/Observable';
+import { Scheduler } from 'rxjs/Scheduler';
 import { Operator } from 'rxjs/Operator';
 
 export declare class ActionsObservable<T> extends Observable<T> {
+  /**
+   * Just like RxJS itself, we can't actually make this method always type-safe
+   * because we would need non-final position spread params e.g.
+   *   `static of<T>(...items: T, scheduler?: Scheduler): ActionsObservable<T>`
+   * which isn't possible in either JavaScript or TypeScript. So instead, we
+   * provide safe typing for up to 6 items, following by a scheduler. 
+   */
+  static of<T>(item1: T, scheduler?: Scheduler): ActionsObservable<T>;
+  static of<T>(item1: T, item2: T, scheduler?: Scheduler): ActionsObservable<T>;
+  static of<T>(item1: T, item2: T, item3: T, scheduler?: Scheduler): ActionsObservable<T>;
+  static of<T>(item1: T, item2: T, item3: T, item4: T, scheduler?: Scheduler): ActionsObservable<T>;
+  static of<T>(item1: T, item2: T, item3: T, item4: T, item5: T, scheduler?: Scheduler): ActionsObservable<T>;
+  static of<T>(item1: T, item2: T, item3: T, item4: T, item5: T, item6: T, scheduler?: Scheduler): ActionsObservable<T>;
+  static of<T>(...array: Array<T | Scheduler>): ActionsObservable<T>;
+
+  static from<T>(ish: ObservableInput<T>, scheduler?: Scheduler): ActionsObservable<T>;
+  static from<T, R>(ish: ArrayLike<T>, scheduler?: Scheduler): ActionsObservable<R>;
```

Comment:
The existing method overrides we have are required because Rx doesn't yet use lift (or something else) to derive from our `ActionsObservable`. These type signature overrides are required because otherwise our method overrides would return the generic `Observable` type instead of `ActionsObservable`, since that's what the inherited definitions return.